readthedocs/api/v2/client.py

```diff
@@ -4,13 +4,12 @@
 
 import requests
 from django.conf import settings
-from requests.packages.urllib3.util.retry import Retry  # noqa
```

Comment:
requests was just importing this from urlib3 https://github.com/psf/requests/blob/master/requests/packages.py